ui/raidboss/data/05-shb/trial/diamond_weapon-ex.txt

```diff
@@ -0,0 +1,116 @@
+### Diamond Weapon Ex
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+# -ii 5FEB 5FBE 5FAA 5FEC 5FBA 5FFD 5FBB 6055 6159 5FF9 5FCD 5FFA 5FF9 612E 5FB3 5FCE 5FD2 5FFE 5FD3 5FB4 5FB6 5FB1 5FB0 5FBF 5FAE 5FA6 5FC3 5FC9 5FCA 5FC4
+
+### Phase 1
+# There are no autoattacks from the boss until after the initial raidwide, so nothing to sync against for autos
+0 "Start" sync /Engage!/ window 1,1
+21.3 "Diamond Rain" sync /:The Diamond Weapon:5FA7:/ window 25,10
+
+# Adamant Purge is a mini-phase that will be either stack, spread, or "nothing"
+# 5FA2 = Spread (Homing Laser, 5FA0)
+# 5F9B = "nothing" (Diamond Rain, 5FA7)
+# 5FA5 = Stack (Diamond Flash, 5FA1)
+# This mechanic repeats in phase 3 with additional bits and an extra tracking If circle aoe
+# @TODO: Someone better at timelines than me, sync these to mini-phases instead
+36.5 "Adamant Purge" sync /:The Diamond Weapon:5F(9[ABCD]|A[2345]):/ window 20,5
+46.7 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+49.9 "Spread/Stack?" sync /:The Diamond Weapon:5FA[01]:/
```

Comment:
I think in general ability names are preferred, and folks can replace with spread/stack in their timelineReplace if they want.  So, `"Homing Laser/Diamond Flash"` instead here.  Also, as a convention, `?` is only used when something may or may not happen.  If it's one or the other, then it's just `A/B` with no `?`.